src/client/modules/my/summaryGraphs/summaryGraphs.ts

```diff
@@ -0,0 +1,59 @@
+import { LightningElement } from 'lwc';
+import { extractPlayerStats, getRawResults } from 'my/resultsFetcher';
+import type { GameResultsData, PlayerStatsAllGames } from 'my/resultsFetcher';
```

Comment:
Some nuance: I separate my `import` and `import type` lines because some compilers can't distinguish, and leave the type imports in JS